src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Lottery {
+    public void getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColorNumber();
+        System.out.println(ball.toString());
```

Comment:
You are using `ball.toString()` to print the ball object, which is not necessary. By default, the `println` method calls the `toString` method of the object, so you can simply pass the ball object to `System.out.println(ball);`. Additionally, ensure that the `toString` method is properly overridden in the `Ball` class to provide a meaningful representation of the ball object.